Auth.org

```diff
@@ -0,0 +1,66 @@
+#+TITLE: Clio Authentication and Authorization
```

Comment:
It just documents my current understanding.
It will wind up in the README, Confluence,
or deleted eventually.